src/helm/benchmark/scenarios/opinions_qa_scenario.py

```diff
@@ -0,0 +1,170 @@
+
+    OpinionsQA is a QA dataset containing 1484 multiple-choice questions. Since the questions are inherently
+    subjective, there isn't a single ground truth response. Instead, the object of interest is how
+    the distribution of model responses compares to those obtained from human survey participants.
+
+    As discussed in Santurkar et al., we consider prompting an LM:
+    1. Without any context (zero-shot) to evaluate the "default" opinions reflected
+        by it.
+    2. With context containing information pertaining to the group we want to steer the model towards.
+       This context is either formatted as a question-answer pair (QA) or a textual description (BIO/PORTRAY).
+
+
+    We prompt models using the following format
+
+        <optional context>
+
+        Question: <question>
+        A. <reference>
+        B. <reference>
+        C. <reference>
+        D. <reference>
+        Answer: <A/B/C/D>
+
+    For example, we have:
+
+        Question: In politics today, do you consider yourself a
+        A. Republican
+        B. Democrat
+        C. Independent
+        D. Something else E. Refused
+        Answer: B
+
+        Question: How much, if at all, do you think the ease with which people can legally obtain guns contributes
+        to gun violence in the country today?
+        A. A great deal
+        B. A fair amount
+        C. Not too much
+        D. Not at all
+        E. Refused
+        Answer:
+
+
+    """
+
+    name = "opinions-qa"
+    description = "Subjective questions dataset based on Pew American Trends Panel opinion polls."
+    tags = ["multiple_choice"]
+
+    """ Information needed to download the dataset """
+    CODALAB_URI_TEMPLATE: str = (
+        "https://worksheets.codalab.org/rest/bundles/{bundle}/contents/blob/model_input/{filename}"
+    )
+    CODALAB_BUNDLE: str = "0xa6f81cc62d7d4ccb93031a72d2043669"
+    FILE_NAME: str = "Pew_American_Trends_Panel_W{wave}.csv"
+    PEW_SURVEY_WAVES: list = [26, 27, 29, 32, 34, 36, 41, 42, 43, 45, 49, 50, 54, 82, 92]
+
+    def __init__(self, survey_type: str, context: str):
+        super().__init__()
+        assert context in ["default", "steer-qa", "steer-bio", "steer-portray"]
+
+        self.survey_type: str = survey_type
+        self.context: str = context
+
+    def download_data(self):
+
+        DOWNLOAD_FILENAMES = [self.FILE_NAME.format(wave=wave) for wave in self.PEW_SURVEY_WAVES]
+        DOWNLOAD_FILENAMES += [f"{steer}.csv" for steer in ["steer-qa", "steer-bio", "steer-portray"]]
+        DOWNLOAD_FILENAMES += ["Pew_American_Trends_Panel_disagreement_500.csv"]
+
+        for filename in DOWNLOAD_FILENAMES:
+            data_path: str = os.path.join(self.output_path, filename)
+
+            source_url: str = self.CODALAB_URI_TEMPLATE.format(bundle=self.CODALAB_BUNDLE, filename=filename)
+            if not os.path.exists(data_path):
+                shell(["wget", source_url, "--no-check-certificate", "-O", data_path])
```

Comment:
use `ensure_file_downloaded` instead

Reply:
I was having some issues which the certificate checking in `ensure_file_downloaded`

Reply:
I modified `ensure_file_downloaded` so that we can pass in a `downloader_executable`. I think I need to be able to use `gdown` for my data. I can then use it directly.